tests/src/test/scala/cats/tests/IdTTests.scala

```diff
@@ -1,7 +1,8 @@
 package cats
 package tests
 
-import cats.data.IdT
+import cats.{Foldable, Functor, Monad, NonEmptyTraverse, Traverse}
```

Comment:
I didn't have the multiple package decls in a previous version of the file, IntelliJ magic.